test/e2e/velero_utils.go

```diff
@@ -292,7 +292,12 @@ func VeleroInstall(ctx context.Context, veleroImage string, veleroNamespace stri
 }
 
 func VeleroUninstall(ctx context.Context, client *kubernetes.Clientset, extensionsClient *apiextensionsclient.Clientset, veleroNamespace string) error {
-	return uninstall.Uninstall(ctx, client, extensionsClient, veleroNamespace)
+	err := uninstall.Run(ctx, client, extensionsClient, veleroNamespace, true)
```

Comment:
Do we want to use that if err := ...; err != nil pattern everywhere?

Reply:
Yes we do.

On a diff note, I don't know why I expanded the error checking. Changed it back to `return uninstall.Uninstall(...`.